src/components/HelloDecorator.vue

```diff
@@ -0,0 +1,39 @@
+<!-- src/components/Hello.vue -->
```

Comment:
Update the comment here, and leave a note about how this is an alternative way to declare the component.